### ISSUE TITLE
Uncontrolled highlighting and underrepresenting font colors 

### ISSUE THREAD

**Issue by Giantmerchant** [ethereum/remix-project#1792]
Bad user experiences.

**Comment by saadqayyum-cyber** [comment]
Problem occured from today. No Syntax highlighting and editor autocompletion. Bad Experience. Switching to VS Code is only option

**Comment by DeezyE** [comment]
Ubuntu 20.04 LTS
Firefox

Can confirm:
- Uncontrolled highlighting that flashes like crazy.
- Shortcut key (ctrl+/) to comment line no longer works
- Extra wide line-count column

**Comment by Astilen** [comment]
Yes, we lost the color cues that we were used to with the dark theme. Now functions are blue and are indistinguishable from variables, and all other elements changed to some other color. Put the old theme in the options even if you dont want to support it please! 

and yes whole code is flashing every time you click somewhere, and you cant tell what you are selecting as everything highlights, it also autoselects the whole line

**Comment by REGO350** [comment]
I totally agree. The new version is super hard to use. And the worse part is that I cannot copy call result anymore!!!

**Comment by LianaHus** [comment]
We did a patch release presently. A lot should be improved already and we are working on farther changes. Unfortunately, we can't reproduce 'Uncontrolled highlighting that flashes'. Is this still an issue for you?

**Comment by whatjakecodes** [comment]
I am still seeing really bad highlighting (using the default Light theme) today:

<img width="1352" alt="image" src="https://user-images.githubusercontent.com/3434442/145289928-1c31d1da-af8e-4d67-a4a3-5dc0fca28923.png">

**Comment by Astilen** [comment]
> We did a patch release recently. A lot should be improved already and we are working on farther changes. Unfortunately, we can't reproduce 'Uncontrolled highlighting that flashes'. Is this still an issue for you?

The problem exists on 5 browsers on windows 10 for me, and its easier to reproduce on a dark theme. Put some comments down and then simply click on different lines, all the the lines blink slightly but the comments are blowing up.

Thank you for adding some colors to the code it really helps out. The function names are still gray, they should pop out with their own unique color, and maybe also the = sign. Thanks!

**Comment by Aniket-Engg** [comment]
Themes are updated. It should be fine now
